### PR TITLE
Add post on desktop

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Forgot link to the forum.

File: _posts/2018-10-03-desktop-update.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Delta.Chat desktop update
+author: magnus
+---
+
+We are working on a desktop application for Delta.Chat. It's written in electron, based on the [node bindings](https://github.com/deltachat/deltachat-node) and has a UI with strong similarities to Signal desktop. There are some early releases for Linux and Mac out already (Windows in the works). You can download them [here](https://github.com/deltachat/deltachat-desktop/releases/).
```
Comment (hpk42) [inline _posts/2018-10-03-desktop-update.md]: it's not just similartieis but partially based on the signal ui desktop code -- that should be clearly mentioned (without claiming it's "based" on laptop security wise or so)

File: _posts/2018-10-03-desktop-update.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+At the moment the application can be considered in early alpha with rudimentary features only. You can:
+
+* login with your IMAP user and password
+* create contacts
+* create chats with contacts
+* send and receive messages (image attachments should work)
+* import keys from another device
+* multi language support (25 languages and counting)
+
```
Comment (hpk42) [inline _posts/2018-10-03-desktop-update.md:18]: doesn't Autocrypt setup  message (syncing between android and desktop) work? not sure if in both directions.

File: _posts/2018-10-03-desktop-update.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* basic testing in the form of:
+  * downloading and installing the application
+  * starting the application
+  * logging in
+  * creating contacts
+  * creating chats with contacts
+  * sending and receiving messages
+* translations via [transifex](https://www.transifex.com/delta-chat/delta-chat-desktop/)
+
```
Comment (hpk42) [inline _posts/2018-10-03-desktop-update.md:39]: maybe also another section on how you can help? (css/design, windows support would be bigger items)

File: _posts/2018-10-03-desktop-update.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+If you have any feedback, please create an account at our [support forum](https://support.delta.chat/). From there we can create issues on respective github repository and take action from there.
```
Comment (hpk42) [inline _posts/2018-10-03-desktop-update.md]: maybe we can create a "desktop" category on the support site and mention it here? 
(don't have much online time today but tomorrow i could help)

**Review by hpk42 — CHANGES_REQUESTED** [review]
looks good, made a few comments regarding clarifications.  can help with finalizing tomorrow.

File: _posts/2018-10-03-desktop-update.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: Delta.Chat desktop update
+author: magnus
```
Comment (hpk42) [inline _posts/2018-10-03-desktop-update.md]: maybe do it as magnus and karissa?

**Comment by r10s** [comment]
apart from the comments from hpk, this looks good to me :)
maybe add a small screenshot?

File: _posts/2018-10-03-desktop-update.md (hunk 2 above)
Comment (ralphtheninja) [reply _posts/2018-10-03-desktop-update.md:18]: Yes, it does. It's what I meant by 'import keys from another device'. Will rephrase.

File: _posts/2018-10-03-desktop-update.md (hunk 4 above)
Comment (ralphtheninja) [reply _posts/2018-10-03-desktop-update.md]: Sure! Let me know when it's created. ping @lefherz

**Comment by ralphtheninja** [comment]
> maybe add a small screenshot?

I'd love to add one, but it's quite a bit of work to get a good screenshot. Have one already? :)

File: _posts/2018-10-03-desktop-update.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+At the moment the application can be considered in early alpha with rudimentary features only. You can:
+
+* login with your IMAP user and password
+* create contacts
+* create chats with contacts
+* create group chats
+* send and receive messages
+* image, audio and video attachments
+* [`AutoCrypt`](https://autocrypt.org/) setup message
```
Comment (hpk42) [inline _posts/2018-10-03-desktop-update.md]: "you can setup message" ? :) 
how about "sending and receiving [Autocrypt setup messages](https://autocrypt.org/level1.html#autocrypt-setup-message) for synchronizing secret keys between devices"

File: _posts/2018-10-03-desktop-update.md (hunk 4 above)
Comment (hpk42) [reply _posts/2018-10-03-desktop-update.md]: no blocker to add this -- quickly checked but the admin of discourse is quite extensive -- not sure where you can add it.  maybe @r10s knows. but we can do this after the post is out.

**Review by hpk42 — APPROVED** [review]
looks good -- i'd change the "autocrypt setup message" wording but otherwise it's fine.

**Comment by ralphtheninja** [comment]
> looks good -- i'd change the "autocrypt setup message" wording but otherwise it's fine.

I just did. You want me to change it again? :)

**Comment by ralphtheninja** [comment]
@r10s this feature:

screenshot from 2018-10-03 12-40-53

**Pull request merged**